go.mod

```diff
@@ -6,7 +6,7 @@ require (
 	github.com/blang/semver v3.5.1+incompatible
 	github.com/boz/go-lifecycle v0.1.1-0.20190620234137-5139c86739b8
 	github.com/caarlos0/env v3.3.0+incompatible
-	github.com/cosmos/cosmos-sdk v0.38.5
```

Comment:
So just a quick question here, is this a tagged release? What is the diff between 0.38.5 and 0.38.6-whatever?

Reply:
The commit `a0fb47f5fd822889ce6b23fb4de884c76a4273d9` lives in the `launchpad/release/v0.39` (it's an RC branch, see https://github.com/cosmos/cosmos-sdk/pull/6707). The odd `0.38.6-` prefix is due to how `go mod` interprets versioning numbers (sort of mix between a semantic versioning-compliant format a `git describe`'s output`. However, I've just tagged the first 0.39.0 release candidate, and I'll update the PR accordingly.